docs/docsite/rst/dev_guide/testing_validate-modules.rst

```diff
@@ -1,12 +1,15 @@
+****************
```

Comment:
We should be encouraging use of validate-modules through `ansible-test sanity`. I'd put instructions for that at the top before the direct usage instructions.